File: Praktikum 2 C/PRAK202-2510817220002-AishaZwarintyanda.c
#include <stdio.h>

int main() {
    float n1, n2;

    printf("Masukkan Nilai Pertama : ");
    scanf("%f", &n1);

    printf("Masukkan Nilai Kedua : ");
    scanf("%f", &n2);

    float total = n1 + n2;

    if (n1 == (int)n1) {
        printf("Hasil dari penjumlahan nilai pertama %d dan nilai kedua %.2f adalah %.2f\n", 
               (int)n1, n2, total);
    } else {
        printf("Hasil dari penjumlahan nilai pertama %.2f dan nilai kedua %.2f adalah %.2f\n", 
               n1, n2, total);
    }

    return 0;
}

File: Praktikum 2 Python/PRAK202-2510817220002-AishaZwarintyanda.py
n1 = float(input("Masukkan nilai pertama : "))
n2 = float(input("Masukkan nilai kedua : "))

total = n1 + n2

if n1.is_integer():
    print("Hasil dari penjumlahan nilai pertama %d dan nilai kedua %.1f adalah %.1f" % (int(n1), n2, total))
else:
    print("Hasil dari penjumlahan nilai pertama %.2f dan nilai kedua %.1f adalah %.2f" % (n1, n2, total))